### ISSUE TITLE
An enhancement to the unlocker

### ISSUE THREAD

**Issue by agoodideaihas** [ItsLogic/Xbox-Achievement-Unlocker#21]
Remove, move or give us the option to disable the "Unlock All Achievements" button.

It makes no sense to unlock all achievements of a game at the same time and it's just next to the "Unlock selected achievements" making everyone a mistake away from the ban hammer. At least add a warning after someone pushes it that it WILL lead to an account ban.

**Comment by ItsLogic** [comment]
> Remove, move or give us the option to disable the "Unlock All Achievements" button.

No. I wont remove functionality from my tool. If you dont want it to show up then compile it yourself with the button removed or disabled. I have no plans to add button enabling and disabling or making a good UI until I am happy with the featureset I provide. This is still very much a work in progress proof of concept project.

> It makes no sense to unlock all achievements of a game at the same time and it's just next to the "Unlock selected achievements" making everyone a mistake away from the ban hammer. At least add a warning after someone pushes it that it WILL lead to an account ban.

Its not guaranteed to get an account ban from clicking the unlock all button nor is it my problem if they click it and get banned. Anyone using a tool like this should be aware of the risks. 

I dont know what to tell you other than just dont click the wrong button. Just look before you click and be careful if you are that paranoid about clicking the wrong one
